aidb/vector_database/marqo_vector_database.py

```diff
@@ -0,0 +1,215 @@
+import uuid
+from dataclasses import dataclass, field
+import numpy as np
+import pandas as pd
+import marqo
+from typing import Dict,  Optional
+
+from aidb.utils.logger import logger
+from aidb.vector_database.vector_database import VectorDatabase
+
+@dataclass
+class MarqoAuth:
```

Comment:
Does Marqo contain these information? When I read the API document, I saw they use mq = marqo.Client(url='http://localhost:8882').

Reply:
I mean does Marqo require username, pwd, api_key?

Reply:
Not for local instance, but yes, it does require for the cloud instance